src/core/server/ui_settings/types.ts

```diff
@@ -52,9 +52,13 @@ export {
  */
 export interface IUiSettingsClient {
   /**
-   * Returns registered uiSettings values {@link UiSettingsParams}
+   * Returns all registered uiSettings values {@link UiSettingsParams}
    */
   getRegistered: () => Readonly<Record<string, PublicUiSettingsParams>>;
+  /**
+   * Returns the overridden uiSettings value if one exists, or the registered default {@link UiSettingsParams}
```

Comment:
Should it be?
```suggestion
   * Returns the overridden uiSettings value if one exists, or the registered default is one exists {@link UiSettingsParams}
```

It looks like the optional chaining of the default value means it can return `undefined`.